p2p/libp2p/netMessenger_test.go

```diff
@@ -1799,3 +1799,29 @@ func TestNetworkMessenger_Bootstrap(t *testing.T) {
 	goRoutinesNumberStart := runtime.NumGoroutine()
 	core.DumpGoRoutinesToLog(goRoutinesNumberStart, log)
 }
+
+func TestLibp2pMessenger_SignVerifyPayloadShouldWork(t *testing.T) {
+	fmt.Println("Messenger 1:")
+	mes1, _ := libp2p.NewNetworkMessenger(createMockNetworkArgs())
+
+	fmt.Println("Messenger 2:")
+	mes2, _ := libp2p.NewNetworkMessenger(createMockNetworkArgs())
+
+	err := mes1.ConnectToPeer(getConnectableAddress(mes2))
+	assert.Nil(t, err)
+
+	defer func() {
+		_ = mes1.Close()
+		_ = mes2.Close()
+	}()
+
+	payload := []byte("payload")
+	sig, err := mes1.Sign(payload)
```

Comment:
can you rename `mes1` and `mes2` to `messenger1` and `messenger2`? `mes` could also be the short form for `message`

Reply:
done in the whole file